recipes/clikit/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "clikit" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ec47b987494320acc0739769f5ae6e092ddba7e0e1a9e2f36c705eabd95a266b
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pastel >=0.1.0,<0.2.0
```

Comment:
Do you need all of these in `host`? `pip`, `python` and `poetry` should be enough